app/api/__init__.py

```diff
@@ -0,0 +1,6 @@
+"""Initializes the api Blueprint."""
+from flask import Blueprint
+
+api = Blueprint('api', __name__)
+
+from . import auth, errors, shorten
```

Comment:
I am not sure if this is avoidable as everything I am importing here are defined within the api Blueprint declared above it. Thanks though